frontend/scss/components/organisms/component-intro.scss

```diff
@@ -125,10 +125,20 @@
           padding-right: 12px;
         }
 
-        &:nth-child(1n + 7) {
+        &:nth-child(1n + 9) {
```

Comment:
Can we reduce the number to 4 on mobile?

Reply:
Sure! :)